src/components/Player.ts

```diff
@@ -0,0 +1,102 @@
+import { GameObj, KaboomCtx } from "kaplay";
+import GameUtils from "../game/GameUtils";
+import GameInstance from "../game/Game";
+
+export default class Player { 
+    public static playerObject: GameObj;
+    private static game: KaboomCtx = GameInstance.getGameInstance().getKObject(); 
+
+    private static playerComponents = [
+      Player.game.sprite('bean'),
+      Player.game.area(), 
+      Player.game.body(), 
+      Player.game.pos(800,500),
+      Player.game.anchor('center'), 
```

Comment:
add the anchor attribute to the `createGameObject` function in the GameObject class